src/app/learn/accessibility/test/index.html

```diff
@@ -0,0 +1,227 @@
+<stache
+ navOrder="3"  
+ navTitle="Testing"  
+ pageTitle="Testing for accessibility"
+ showTableOfContents="true">
+  
+  <stache-page-summary>
+    <p>
+      SKY UX employs several methods to test components for accessibility. You can use the same practices and tools to ensure your applications are highly accessible.
+    </p>
+  </stache-page-summary>
+  
+  <stache-page-anchor>
+    Run automated accessibility tests
+  </stache-page-anchor>
+  
+  <p>
+    SKY UX incorporates <a href="https://github.com/dequelabs/axe-core">aXe automated tests</a> into its <a routerLink="/learn/get-started/advanced/accessibility-analyzer">accessibility analyzer</a> for use in end-to-end(e2e) tests. aXe includes 50-60 rules covering a subset of the W3C's WCAG 2.0 success criteria.
+  </p>
+  <p>
+    Include accessibility checks in your e2e tests using the  <a routerLink="/learn/get-started/advanced/accessibility-analyzer">accessibility analyzer</a>, and ensure that all the rules pass.
+  </p>
+  
+  <stache-page-anchor>
+    Incorporate accessibility checks in a code review
+  </stache-page-anchor>
+  
+  <p>
+    Perform a review using the following SKY UX accessibility code review checklist to identify potential accessibility problems that automated aXe testing can not evaluate.
+  </p>
+  <p>
+    Complete the following checks when reviewing your application's markup code along with the user interface:
+  </p>
+  
+  <h3>
+    Text equivalents for non-text elements
+  </h3>
+  
+  <ol>
+    <li>
+      <p>
+        Ensure that <stache-code>alt</stache-code> attributes on <stache-code>&lt;img&gt;</stache-code> elements accurately reflect the content and/or function of the image in the <a href="http://webaim.org/techniques/alttext/#context">context that the image</a> is used.
+      </p>
+    </li>
+    <li>
+      <p>
+        Ensure that button text values accurately describe the purpose of buttons.
+      </p>
+    </li>
+    <li>
+      <p>
+        Ensure that any decorative images have the <stache-code>role="presentation"</stache-code> attribute.
+      </p>
+    </li>
+  </ol>
+  
+  <h3>
+    Page order
+  </h3>
+  
+  <p>
+    Ensure that the visual order of elements matches the DOM order of elements.
+  </p>
+  
```

Comment:
@Blackbaud-MattGregg, in the current docs, we have a "Link purpose" section between the "Order" and "Semantic purpose" sections. I don't see that content anywhere here. Am I missing it? If not, did we exclude it intentionally?